modules/installation-and-upgrade/pages/container-deployment/mlm/proxy-deployment-mlm.adoc

```diff
@@ -120,66 +126,122 @@ transactional-update
 
 . Reboot the system.
 . Log in as root.
-. Install the container utilities:
+. Install the container utilities ([package]``mgrpxy`` should already be installed automatically):
+
 +
+
 [source, shell]
 ----
-transactional-update pkg install mgrpxy mgrpxy-bash-completion
+transactional-update pkg install mgrpxy-bash-completion
 ----
+
 +
-* Alternatively you may install [systemitem]``mgrpxy-zsh-completion`` or [systemitem]``mgrpxy-fish-completion``.
+
+Alternatively you may install [systemitem]``mgrpxy-zsh-completion`` or [systemitem]``mgrpxy-fish-completion``.
+
 +
+
 . Reboot the system.
 
+[IMPORTANT]
+====
+{sl-micro} is designed to update itself automatically by default and will reboot after applying updates.
+However, this behavior is not desirable for the {productname} environment.
+To prevent automatic updates on your server, {productname} disables the transactional-update timer during the bootstrap process.
+
+If you prefer the {sl-micro} default behavior, enable the timer by running the following command:
+
+[source, shell]
+----
+systemctl enable --now transactional-update.timer
+----
+====
+
+To continue with deployment, see xref:installation-and-upgrade:container-deployment/mlm/proxy-deployment-mlm.adoc#deploy-mlm-proxy-persistent-storage[].
+
 
 
-== Deploy {productname} Proxy on {sles} {bci-mlm}
+[[deploy-mlm-proxy-sles]]
+=== Install {sles} {bci-mlm} and Deploy Proxy
```

Comment:
I would prefer not to include `Deploy Proxy` words at all here.
This is really about installing (or if you must deploying) proxy host.

Actual proxy is deployed on step 9.

Reply:
I created a new issue to find better wording.